Source/WebKitLegacy/mac/Misc/WebDownload.mm

```diff
@@ -225,12 +225,14 @@ - (void)_setRealDelegate:(id)delegate
 - (id)init
 {
     self = [super init];
-    if (self != nil) {
-        // _webInternal can be set up before init by _setRealDelegate
-        if (_webInternal == nil) {
-            _webInternal = [[WebDownloadInternal alloc] init];
-        }
+    if (self == nil)
+        return nil;
+
+    // _webInternal can be set up before init by _setRealDelegate
+    if (_webInternal == nil) {
```

Comment:
As you are touching these lines anyway, please remove braces around the single line block.

Reply:
Addressed!